src/ClusterBootstrap/services/logging/elasticsearch.yaml

```diff
@@ -1,71 +1,117 @@
     metadata:
       labels:
-        k8s-app: elasticsearch-logging
-        version: v5.6.2
-        kubernetes.io/cluster-service: "true"
+        k8s-app: elasticsearch
+      annotations:
+        prometheus.io/scrape: 'true'
+        prometheus.io/path: '/metrics'
+        prometheus.io/port: '{{cnf["elasticsearch"]["port"]["exporter"]}}'
     spec:
       hostNetwork: true
       nodeSelector:
         elasticsearch: active
+      initContainers:
+      - name: elasticsearch-sysctl
+        image: alpine:3.9
+        command:
+        - "/sbin/sysctl"
+        - "-w"
+        - "vm.max_map_count=262144"
+        securityContext:
+          privileged: true
+      - name: elasticsearch-chown
+        image: alpine:3.9
+        command:
+        - "chown"
+        - "-R"
+        - "1000:1000"
+        - "/usr/share/elasticsearch/data"
+        volumeMounts:
+        - name: elasticsearch
+          mountPath: /usr/share/elasticsearch/data
       containers:
-      - image: mlcloudreg.westus.cloudapp.azure.com:5000/dlworkspace/elasticsearch
-        imagePullPolicy: Always
-        name: elasticsearch-logging
+      - name: elasticsearch
+        image: docker.elastic.co/elasticsearch/elasticsearch:6.8.5
+        env:
+        - name: "node.name"
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        - name: "discovery.zen.ping.unicast.hosts"
+          value: '
+            {%- for elasticsearch_node in cnf["elasticsearch_node"] -%}
+            {{elasticsearch_node}}{{"," if not loop.last}}
+            {%- endfor -%}
+          '
+        - name: "http.port"
+          value: '{{cnf["elasticsearch"]["port"]["http"]}}'
+        - name: "transport.port"
+          value: '{{cnf["elasticsearch"]["port"]["transport"]}}'
         resources:
           # need more cpu upon initialization, therefore burstable class
           limits:
             cpu: 1000m
           requests:
             cpu: 100m
         ports:
-        - containerPort: {{cnf["elasticsearch_db_port"]}}
-          name: db
+        - containerPort: {{cnf["elasticsearch"]["port"]["http"]}}
+          name: http
           protocol: TCP
-        - containerPort: {{cnf["elasticsearch_tp_port"]}}
+        - containerPort: {{cnf["elasticsearch"]["port"]["transport"]}}
           name: transport
           protocol: TCP
         volumeMounts:
-        - name: elasticsearch-logging
-          mountPath: /data
-        env:
-        - name: "NAMESPACE"
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.namespace
-        - name: "MINIMUM_MASTER_NODES"
-          value: "1"
-        - name: "HTTP_PORT"
-          value: "{{cnf["elasticsearch_db_port"]}}"
-        - name: "TRANSPORT_PORT"
-          value: "{{cnf["elasticsearch_tp_port"]}}"
+        - name: elasticsearch
+          mountPath: /usr/share/elasticsearch/data
+      - name: elasticsearch-exporter
+        image: justwatch/elasticsearch_exporter:1.1.0
```

Comment:
please change the docker image to configurable parameter. Default value can be found in params.py